cognite/client/data_classes/transformation_jobs.py

```diff
@@ -0,0 +1,249 @@
+
+
+class TransformationJob(CogniteResource):
+    """The transformation job resource allows following the status of execution of a transformation run.
+
+    Args:
+        id (int): A server-generated ID for the object.
+        uuid (UUID): A server-generated UUID for the object.
+        status (TransformationJobStatus): Status of the job.
+        transformation_id (int): Server-generated ID of the transformation.
+        source_project (str): Name of the CDF project the data will be read from.
+        destination_project (str): Name of the CDF project the data will be written to.
+        destination_type (str): Target resource type of the transformation.
+        destination_database (str): Target database if the destination type is raw.
+        destination_table (str): Target table name if the destination type is RAW.
+        conflict_mode (str): What to do in case of id collisions: either "abort", "upsert", "update" or "delete".
+        raw_query (str): Query of the transformation that is being executed.
+        error (str): Error message from the server.
+        ignore_null_fields (bool): Indicates how null values are handled on updates: ignore or set null.
+        created_time (int): Time when the job was created.
+        started_time (int): Time when the job started running.
+        finished_time (int): Time when the job finished running.
+        last_seen_time (int): Time of the last status update from the job.
+        cognite_client (CogniteClient): The client to associate with this object.
+    """
+
+    def __init__(
+        self,
+        id: int = None,
+        uuid: UUID = None,
+        status: TransformationJobStatus = None,
+        transformation_id: int = None,
+        source_project: str = None,
+        destination_project: str = None,
+        destination: "TransformationDestination" = None,
+        conflict_mode: str = None,
+        raw_query: str = None,
+        error: str = None,
+        ignore_null_fields: bool = False,
+        created_time: int = None,
+        started_time: int = None,
+        finished_time: int = None,
+        last_seen_time: int = None,
+        cognite_client=None,
+    ):
+        self.id = id
+        self.uuid = uuid
+        self.status = status
+        self.transformation_id = transformation_id
+        self.source_project = source_project
+        self.destination_project = destination_project
+        self.destination = destination
+        self.conflict_mode = conflict_mode
+        self.raw_query = raw_query
+        self.error = error
+        self.ignore_null_fields = ignore_null_fields
+        self.created_time = created_time
+        self.started_time = started_time
+        self.finished_time = finished_time
+        self.last_seen_time = last_seen_time
+        self._cognite_client = cognite_client
+
+    def update(self):
+        """`Get updated job status. <https://docs.cognite.com/api/playground/#operation/runTransformation>`_"""
```

Comment:
don't need this here I think. it's just a data class.

Reply:
same for other methods.

Reply:
instead should explain the args.

Reply:
you're right, since they don't correspond to a specific api call, no link is necessary.